### PR TITLE
os/fs/driver/mtd : Modify journaling object structures

### PR DESCRIPTION
**Pull request by amogh-samsung**
- change struct smart_journal_logging_data_s ->
  smart_journal_entry_s.
- change njournaldata -> njournalentries in struct
  smart_struct_s

Signed-off-by: Amogh Hassija <a.hassija@samsung.com>

### CODE REVIEW

File: os/fs/driver/mtd/smart.c [hunk 1]
```diff
@@ -304,9 +304,9 @@ struct smart_struct_s {
 #endif
 #ifdef CONFIG_MTD_SMART_JOURNALING
 	size_t journal_seq;				/* Current Sequence of Journal */
-	uint16_t njournalPerBlk;		/* Total Number of Journal data per Erase block */
-	uint16_t njournaleraseblocks;	/* Total Number of Journal Erase block */
-	uint32_t njournaldata;			/* Total Number of Journal Data */
+	uint16_t njournalPerBlk;		/* Total Number of Journal entries per Erase block */
+	uint16_t njournaleraseblocks;		/* Total Number of Journal Erase block */
+	uint32_t njournalentries;			/* Total Number of Journal Entries */
```
Comment (deepaksrma) [inline os/fs/driver/mtd/smart.c:309]: could you maintain the same tab space for comments

File: os/fs/driver/mtd/smart.c [hunk 2]
```diff
@@ -1041,7 +1041,7 @@ static int smart_setsectorsize(FAR struct smart_struct_s *dev, uint16_t size)
 	
 #ifdef CONFIG_MTD_SMART_JOURNALING
 	/** Journal Sector is reserved at the last of smartfs partition, it doesn't use MTD Header.
-	  * Journal sectors divided to 2 area, this will be toggled if anyone is filled with data,
+	  * Journal sectors divided to 2 area, this will be toggled if one area is filled with data,
```
Comment (Taejun-Kwon) [inline os/fs/driver/mtd/smart.c:1044]: we don't use area now, could you please remove this description? I forgot it.

**Comment by Taejun-Kwon** [comment]
Please remove 'If fs->entry_seq is same as timestamp, it doesn't increase number, so fix it' from PR description.
Is 'entry' is more comfortable than '(transaction)logging data' ?

File: os/fs/driver/mtd/smart.c (hunk 1 above)
Comment (amogh-samsung) [reply os/fs/driver/mtd/smart.c:309]: Ok

File: os/fs/driver/mtd/smart.c (hunk 2 above)
Comment (amogh-samsung) [reply os/fs/driver/mtd/smart.c:1044]: Ohkay

**Pull request merged**